modules/generic_worker/templates/run-generic-worker.sh.erb

```diff
@@ -32,7 +32,7 @@ exec 2> >(tee /var/log/genericworker/stderr.log >&2)
 exitstatus="${PIPESTATUS[0]}"
 
 # Remove semaphore file to ensure generic worker does not start before puppet
-rm -f "/var/tmp/semaphore/run-buildbot"
+#rm -f "/var/tmp/semaphore/run-buildbot"
```

Comment:
This is needed because for some reason we hit that rm intermittently, which causes workers to stop.
This was for puppet-on-boot which we don't have set up anyway. There are two pools running with this change and they work fine.